### PR TITLE
Library to manage a task that is broken into sequential subtasks.

### PR DESCRIPTION
**Pull request by ttung**
See the documentation.

### CODE REVIEW

File: tests/test_parallel_worker.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss.util import parallel_worker
+from tests.infra import testmode
+
+
+class TestParallelWorker(unittest.TestCase):
+    @testmode.standalone
+    def test_sequential_complete(self, subtasks=5):
```
Comment (Bento007) [inline tests/test_parallel_worker.py]: what if subtask is 0?

Comment (ttung) [reply tests/test_parallel_worker.py]: I don't understand your question. :(

Comment (Bento007) [reply tests/test_parallel_worker.py]: will everything work if `test_sequential_complete(subtasks=0)`

Comment (Bento007) [reply tests/test_parallel_worker.py]: add that test and i'll approve

**Comment by ttung** [comment]
Added test for empty list of incomplete tasks.

File: tests/test_parallel_worker.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+from tests.infra import testmode
+
+
+class TestParallelWorker(unittest.TestCase):
+    @testmode.standalone
+    def test_empty_tasklist(self):
+        """Test the case where the tasklist is empty."""
+        reporter = RecordingReporter()
+        runner = parallel_worker.Runner(8, LatchedTask, [], reporter)
+        runner.run()
```
[the remark this answers is not in the document]
Comment (ttung) [reply tests/test_parallel_worker.py]: done.

**Pull request merged**